api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/dataset/definition/SharedDataDefintion.java

```diff
@@ -36,4 +54,45 @@ public DataDefinition definition(String name, Concept concept) {
         obsForPersonDataDefinition.setWhich(TimeQualifier.LAST);
         return obsForPersonDataDefinition;
     }
+    public PersonAttributeType getPhoneNumber() {
+        PersonAttributeType phoneNumber = Context.getPersonService().getPersonAttributeTypeByUuid("14d4f066-15f5-102d-96e4-000c29c2a5d7");
+        return phoneNumber;
+    }
+    PatientIdentifierType preARTNo = MetadataUtils.existing(PatientIdentifierType.class, "e1731641-30ab-102d-86b0-7a5022ba4115");
+
+    public DataDefinition getNationalIdentifiernumber()
+    {
+        PatientIdentifierType nationalIdentifiernumber = MetadataUtils.existing(PatientIdentifierType.class, "f0c16a6d-dc5f-4118-a803-616d0075d282");
+        DataDefinition nationalID = new ConvertedPatientDataDefinition("identifier", new PatientIdentifierDataDefinition(nationalIdentifiernumber.getName(), nationalIdentifiernumber), identifierFormatter);
+        return nationalID;
+}
+
+    DataConverter identifierFormatter = new ObjectFormatter("{identifier}");
```

Comment:
@solemabrothers I do not understand this code which is outside functions

Reply:
Removed